### ISSUE TITLE
Provide example YAML dashboard

### ISSUE THREAD

**Issue by andreondra** [andreondra/homeassistant-sermatec-inverter#28]
We should provide an example dashboard yaml config. Something like [this](https://github.com/tahvane1/jablotron80#example-control-panel).

**Comment by mathieupotier** [comment]
![image](https://github.com/sermatec-opensource/homeassistant-sermatec-inverter/assets/10141454/5a1ef065-cd6e-4534-9a39-4fbafe149bd2)

This kind of things ?

**Comment by andreondra** [comment]
Yes! This looks really cool. However, Home Assistant now generates entity IDs automatically and I wasn't able to set it manually in the integration, so it is not possible right now to make an universal example for all languages.
